### PR TITLE
Report a low-confidence msg < certain threshold

### PR DESCRIPTION
**Pull request by pradh**
image

### CODE REVIEW

File: server/lib/nl/detection/utils.py [hunk 1]
```diff
@@ -96,6 +96,14 @@ def get_multi_sv_pair(
   return parts
 
 
+def get_top_sv_score(detection: Detection) -> float:
+  if is_multi_sv(detection):
```
Comment (jehangiramjad) [inline server/lib/nl/detection/utils.py:100]: the return type is a float so shouldn't this return something? else, the return type hint could be a Union[float, None] or something like that?

**Review by jehangiramjad — APPROVED** [review]
Thanks!

File: server/lib/nl/detection/utils.py (hunk 1 above)
Comment (pradh) [reply server/lib/nl/detection/utils.py:100]: Sorry, not following the Q, it is returning the score, or 0?

Comment (pradh) [reply server/lib/nl/detection/utils.py:100]: I'll merge now and handle something I missed based on your response

Comment (jehangiramjad) [reply server/lib/nl/detection/utils.py:100]: Ooops sorry. Too late in the night. My screen was showing me a line break so it seemed like it was just a dangling return statement... my bad!

**Pull request merged**